quick_purchase/README.rst

```diff
@@ -0,0 +1,52 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==============
+Quick Purchase
+==============
+This module gives the possibility
+to add easily product into the purchase order (mass line add/update).
+
```

Comment:
Could you add more details on use cases, which kind of process should I use your module ?

If i have 10000...000 products is it an advantage to use it or not ? A lot of config on products !